basic/src/test/resources/jboss-ejb-client.properties

```diff
@@ -0,0 +1,30 @@
+#
+# JBoss, Home of Professional Open Source.
+# Copyright 2011, Red Hat, Inc., and individual contributors
+# as indicated by the @author tags. See the copyright.txt file in the
+# distribution for a full listing of individual contributors.
+#
+# This is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; either version 2.1 of
+# the License, or (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this software; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+#
+remote.connectionprovider.create.options.org.xnio.Options.SSL_ENABLED=false
+
+remote.connections=jbossts
+
+remote.connection.jbossts.host=${host.url:localhost}
+remote.connection.jbossts.port=8080
+remote.connection.jbossts.connect.options.org.xnio.Options.SASL_POLICY_NOANONYMOUS=false
+remote.connection.jbossts.connect.options.org.xnio.Options.SASL_POLICY_NOPLAINTEXT=false
+
```

Comment:
As this is more general configuration then for transactions, I would suggest using different name for remote.connections to make it clear it is usable also for other purposes where remote ejb calls are needed. But OK.